docs/Interoperator-Features.md

```diff
@@ -78,4 +78,21 @@ As a Service Provider, if one wants to define quotas on services provisioned, It
 
   * Already used quota is calculated from the APIServer and does not need any external dependency like CF.
 
-  * Remaining quota is calculated based on above two values and based on that provision request is allowed.
\ No newline at end of file
+  * Remaining quota is calculated based on above two values and based on that provision request is allowed.
+
+## Asynchronous Service Binding Operations
+
+If the service supports, it is possible to perform service binding operations asynchronously. To leverage asynchronous service binding for the plans, under SFPlans, one needs to add the following metadata.
+
+```
+spec
+  manager:
+  async: true
+  asyncBinding: false
+
+```
+
+If `asyncBinding` is set to `true` for the plan, the service binding operation will be asynchronous. The default value is `false` if omitted from the plan and the default behavior is synchronous. The `async` flag is used to control the behavior of service provisioning operations.

```

Comment:
will be asynchronous if the platform passes the `accept_incomplete` flag as `true`

Reply:
Added.